pkg/apis/enterprise/v1alpha1/splunkenterprise_types.go

```diff
@@ -14,20 +24,22 @@ import (
 // NOTE: Do not make struct fields private (i.e. make sure all struct fields start with an uppercase) otherwise the
 //		 serializer will not be able to access those fields and they will be left blank
 
+// SplunkResourcesSpec defines the resource requirements for a SplunkEnterprise deployment
 type SplunkResourcesSpec struct {
-	SplunkCpuRequest     string `json:"splunkCpuRequest"`
-	SparkCpuRequest      string `json:"sparkCpuRequest"`
+	SplunkCPURequest     string `json:"splunkCpuRequest"`
```

Comment:
This is definitely a future-improvement thing, but one thing I've seen as a standard on most helm charts is a completely empty block for `resources` which users can add their own map of limits/requests as you would see on a regular `Pod` spec - might be something we want to consider instead of managing 4 variables like this. Example: https://github.com/helm/charts/blob/master/stable/prometheus/values.yaml#L239

I know this isn't helm, but it may be a more flexible system - especially when future requests come in for tolerations/nodeSelector/affinity/annotations/etc.

Reply:
Yes, plan is to ultimately break up this CRD into more specific component CRDs and re-work all these resource configs at that time.